src/GoTrueApi.ts

```diff
@@ -215,6 +216,32 @@ export default class GoTrueApi {
     }
   }
 
+  /**
+   * Logs in an OpenID Connect user using their id_token.
+   * @param IDToken The IDToken of the user.
+   * @param nonce The nonce of the user. The nonce is a random value generated by the developer (= yourself) before the initial grant is started. You should check the OpenID Connect specification for details. https://openid.net/developers/specs/
+   * @param provider The provider of the user.
+   * @param clientID The clientID of the user.
+   * @param issuer The issuer of the user.
+  */
+  async signInWithOpenIDConnect({ IDToken, nonce, clientID, issuer, provider }:OpenIDConnectCredentials): Promise<{ data: Session | null; error: ApiError | null }> {
+    try {
+      const headers = { ...this.headers }
+      const queryString = '?grant_type=id_token'
+      const data = await post(
+        this.fetch,
+        `${this.url}/token${queryString}`,
+        { id_token: IDToken, nonce, client_id: clientID, issuer, provider },
```

Comment:
Could one reuse `id_token` and `client_token` instead of `IDToken` and `clientID` so that these are always consistent?

Reply:
Changed.